openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -13,40 +13,40 @@ <h2 class="title">
     </h2>
 
     <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
-        <div class="ui-staff ui-toggle-visibility is--collapsed">
-            <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+        <button class="ui-staff ui-slidable" aria-expanded="false" id="learner_response_{{ submission.uuid }}">
```

Comment:
Note to reviewers: just always use a button! The id is being used to link the button with its associated content, since it is less clear here than in the other places that "toggle" buttons are used. I'm using the submission.uuid to ensure unique IDs should there ever be 2 instances on the page at once.